### PR TITLE
App config

### PR DESCRIPTION
**Pull request by nickdala**

### CODE REVIEW

File: infra/shared/terraform/modules/acr/main.tf [hunk 1]
```diff
@@ -96,7 +96,7 @@ resource "azurerm_private_dns_zone_virtual_network_link" "virtual_network_link_a
   count                 = var.environment == "prod" ? 1 : 0
   name                  = "privatelink.azurecr.io"
   private_dns_zone_name = azurerm_private_dns_zone.dns_for_acr[0].name
-  virtual_network_id    = var.spoke_vnet_id
+  virtual_network_id    = var.hub_vnet_id
```
Comment (mnriem) [inline infra/shared/terraform/modules/acr/main.tf:99]: Why is it moving?

File: infra/shared/terraform/modules/service-bus/main.tf [hunk 2]
```diff
@@ -29,8 +29,6 @@ resource "azurerm_servicebus_namespace" "servicebus_namespace" {
   # https://github.com/hashicorp/terraform-provider-azurerm/issues/26570
   local_auth_enabled = true
 
-  zone_redundant = var.environment == "prod" ? true : false
-
```
Comment (mnriem) [inline infra/shared/terraform/modules/service-bus/main.tf:33]: Is it no longer zone redundant?

**Review by mnriem — APPROVED** [review]

File: infra/shared/terraform/modules/acr/main.tf (hunk 1 above)
Comment (nickdala) [reply infra/shared/terraform/modules/acr/main.tf:99]: This aligns to the architecture for Enterprise App Patterns.  The dotnet sample also has the container registry in the hub https://github.com/Azure/modern-web-app-pattern-dotnet/blob/dfddc71a6e5558618b31699944009d844f511246/infra/modules/hub-network.bicep#L285

File: infra/shared/terraform/modules/service-bus/main.tf (hunk 2 above)
Comment (nickdala) [reply infra/shared/terraform/modules/service-bus/main.tf:33]: No longer needed as it's enabled by default.

https://registry.terraform.io/providers/hashicorp/azurerm/3.116.0/docs/resources/servicebus_namespace#zone_redundant

**Pull request merged**